### DISCUSSION TITLE
the eval is not working like -eval message.member.voice.join() and message.guild.voice.setDeaf(true)

### DISCUSSION THREAD

**Discussion by suptrpls** [jagrosh/MusicBot#729] [Questions] [answered]
its not rlly working

**Comment by jagrosh** [comment, marked as answer]
Those are not valid evaluation statements. Please keep in mind that JMusicBot's eval command uses the [Nashorn engine](https://docs.oracle.com/javase/9/nashorn/introduction.htm#JSNUG136) and the [JDA library](https://github.com/DV8FromTheWorld/JDA) for Discord-related internals. 

You will likely need to be proficient with both Nashorn and JDA before attempting to use the eval command.

**Reply by suptrpls** [reply]
ok thx

**Reply by suptrpls** [reply]
one more Question can i host this javascript on vultr
